toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -660,3 +723,95 @@ func getPartUUID(device string) (uuid string, err error) {
 	uuid = strings.TrimSpace(stdout)
 	return
 }
+
+func checkSectorSizeFile(sectorFile string) (sectorSize uint64, err error) {
```

Comment:
>checkSectorSizeFile [](http://example.com/codeflow?start=5&length=19)

I think the previous `getSectorSize` name might have been better aligned with what the function does, since we still return sector size. Alternatively `readSectorSize` or similar.

Reply:
Fixed